src/js/letters/utils/helpers.jsx

```diff
@@ -292,66 +301,27 @@ export function getStateName(stateCode) {
   return stateName || '';
 }
 
-// FOR TESTING PURPOSES ONLY; DO NOT LET THIS INTO MASTER
-// export function getAddressSuccessAction() {
-//   return {
-//     type: 'GET_ADDRESS_SUCCESS',
-//     data: {
-//       data: {
-//         id: '',
-//         type: 'evss_letters_letters_responses',
-//         attributes: {
-//           letters: [
-//             {
-//               name: 'Commissary Letter',
-//               letterType: 'commissary'
-//             },
-//             {
-//               name: 'Proof of Service Letter',
-//               letterType: 'proof_of_service'
-//             },
-//             {
-//               name: 'Proof of Creditable Prescription Drug Coverage Letter',
-//               letterType: 'medicare_partd'
-//             },
-//             {
-//               name: 'Proof of Minimum Essential Coverage Letter',
-//               letterType: 'minimum_essential_coverage'
-//             },
-//             {
-//               name: 'Service Verification Letter',
-//               letterType: 'service_verification'
-//             },
-//             {
-//               name: 'Civil Service Preference Letter',
-//               letterType: 'civil_service'
-//             },
-//             {
-//               name: 'Benefit Summary Letter',
-//               letterType: 'benefit_summary'
-//             },
-//             {
-//               name: 'Benefit Verification Letter',
-//               letterType: 'benefit_verification'
-//             }
-//           ],
-//           fullName: 'Gary Todd',
-//           address: {
-//             addressOne: '123 Main St N',
-//             addressTwo: 'Apt B',
-//             addressThree: 'Attn: That Guy',
-//             type: 'DOMESTIC',
-//             city: 'Central City',
-//             state: 'OR',
-//             country: 'USA',
-//             zipCode: '55555'
-//           },
-//           controlInformation: {
-//             canUpdate: true
-//           }
-//         }
-//       }
-//     }
-//   };
-// }
+/**
+ * Infers the address type from the address supplied and returns the address
+ *  with the "new" type.
+ */
+export function inferAddressType(address) {
+  let type = ADDRESS_TYPES.domestic;
+  if (address.countryName !== 'USA') {
+    type = ADDRESS_TYPES.international;
+  } else if (MILITARY_STATES.has(address.stateCode)) {
+    type = ADDRESS_TYPES.military;
+  }
+
+  return Object.assign({}, address, { type });
+}
 
+export function resetDisallowedAddressFields(address) {
```

Comment:
Would you mind adding a comment before this function explaining why it's necessary, that we don't pass state and zip for international addresses so we'd want to clear those out? I think that would be helpful documentation to add here, thanks!